hal/riscv64/pmap.c

```diff
@@ -342,14 +347,12 @@ char pmap_marker(page_t *p)
 /* Function initializes low-level page mapping interface */
 void _pmap_init(pmap_t *pmap, void **vstart, void **vend)
 {
-	void *v;
 	struct {
 		u64 addr;
 		u64 limit;
 	} *m, *r;
 	size_t n, i;
 	u64 a, l;
-	u64 e = (u64)_end;
```

Comment:
I also meant `e` variable

Reply:
Yeah, I thought that it is unused so I removed it.